src/foam/u2/detail/SectionView.js

```diff
@@ -53,13 +53,21 @@ foam.CLASS({
 
       self
         .addClass(self.myClass())
-        .add(self.slot(function(section, showTitle, section$title) {
+        .add(self.slot(function(section, showTitle, section$title, section$subTitle) {
           if ( ! section ) return;
           return self.Rows.create()
             .show(section.createIsAvailableFor(self.data$))
             .callIf(showTitle && section$title, function() {
               this.start('h2').add(section$title).end();
             })
+            .callIf(section$subTitle, function() {
+              this.start().style({
+                'color': '/*%GREY4%*/ #8e9090',
```

Comment:
Does variable substitution work when you inline it? If not, better to add to css: block.